packages/conf-adwaita-icon-theme/conf-adwaita-icon-theme.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "7895506+MSoegtropIMC@users.noreply.github.com"
+homepage: "https://github.com/adwaita/adwaita-icon-theme"
+bug-reports: "https://gitlab.adwaita.org/adwaita/adwaita-icon-theme/issues"
+authors: "GNU Project"
+license: ["LGPL-3.0-only" "CC-BY-SA-3.0"]
+# E.g. on Fedora the system package does not contain a .pc file.
+# Testing for this folder seems to be more portable than using system packager commands as fallback.
+build: [ "sh" "-c" "pkg-config --short-errors --print-errors adwaita-icon-theme || [ -d /usr/share/icons/Adwaita ]"]
+depends: [
+  "conf-pkg-config" {build}
+]
+depexts: [
+  # The list of supported os-family names can be derived from the opam sources
+  #   https://github.com/ocaml/opam/blob/6aefe95e60084e63d01b1c7c028b6b77de1f839f/src/state/opamSysInteract.ml#L84-L123
+  # The package name for many Linux/BSD/macOS/cygwin/msys platforms can be searched here
+  #   https://repology.org specifically https://repology.org/project/adwaita-icon-theme/versions
+  #   Not listed above but supported by opam are suse, oraclelinix, ol, rhel
+  [ "adwaita-icon-theme" "adwaita-icon-theme-dev" ] {os-family = "alpine"}
+  [ "adwaita-icon-theme" ] {os-family = "amzn"}
+  [ "adwaita-icon-theme" ] {os-family = "arch"}
+  [ "adwaita-icon-theme" ] {os-family = "archlinux"}
+  [ "adwaita-icon-theme" ] {os-family = "centos"}
+  [ "adwaita-icon-theme" ] {os-family = "debian"}
+  [ "adwaita-icon-theme" ] {os-family = "fedora"}
+  [ "x11-themes/adwaita-icon-theme" ] {os-family = "gentoo"}
+  [ "adwaita-icon-theme" ] {os-family = "homebrew"}
+  [ "adwaita-icon-theme" ] {os-family = "macports"}
```

Comment:
Don't you need `{os = "macos" & os-distribution = "macports"}` and `{os = "macos" & os-distribution = "homebrew"}` here?

Reply:
I would think that the os-families homebrew and macports are unique. I think one should use the os-families as most generic classes unless one needs to further break it down - I guess this is the reason d'être for the OS families. So I looked up in the opam sources what exists and went from there. But quite possibly this is a missunderstading on my side.

The package names I get from repology - as stated in the comment. I actually test on Macports, Ubuntu 18 and 16 and Fedore 32.